setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Function to download a file if it does not exist
+download_file() {
+  local url=$1
+  local target_path=$2
+
+  # Check if the file already exists to avoid re-downloading
+  if [ ! -f "$target_path" ]; then
+    echo "Downloading $(basename "$target_path")..."
+    curl -s -L "$url" -o "$target_path" || { echo "Failed to download $(basename "$target_path"). Exiting."; exit 1; }
+  else
+    echo "$(basename "$target_path") already exists. Skipping download."
+  fi
+}
+
+# Set databases path
+GRIDSUITE_DATABASES=$(realpath "DATABASES")
+export GRIDSUITE_DATABASES
```

Comment:
We don't support a "DATABASES" env var, we define directly "GRIDSUITE_DATABASES" in our env (`.bashrc` for example).
It's up to users to how they manage it in their environment.

Reply:
here it creates default folder DATABASES, it is not an ENV variable, the ENV variable is still "GRIDSUITE_DATABASES". One could add check, if it is already defined, then it will not be set again